tests/test_subscriptions.py

```diff
@@ -227,6 +233,15 @@ def _put_subscription(self, endpoint=None, expect_code=None, attachments=None):
         )
         return resp_obj.json.get('uuid')
 
+    def _cleanup_subscription(self, uuid, subscription_type=None):
+        if not subscription_type:
+            subscription_type = 'elasticsearch'
+        url = (UrlBuilder()
+               .set(path=f"/v1/subscriptions/{uuid}")
+               .add_query("replica", self.replica.name)
+               .add_query('subscription_type', subscription_type))
+        self.assertDeleteResponse(url, requests.codes.okay, headers=get_auth_header())
```

Comment:
If you use addCleanup don't thrown an assertion in the cleanup, otherwise the rest of the teardown will fail.

Reply:
ah, i dont like that, i wanna test that they all get deleted, if they are not deleting thats an issue, ill move back to the commit with `try: finally:`